packages/db/src/schema/target-group.ts

```diff
@@ -11,6 +13,17 @@ export const targetMetadataGroup = pgTable("target_metadata_group", {
   name: text("name").notNull(),
   description: text("description").notNull(),
   keys: text("keys").array().notNull(),
+  includeNullCombinations: boolean("include_null_combinations")
+    .notNull()
+    .default(false),
 });
 
+export const createTargetMetadataGroup = createInsertSchema(targetMetadataGroup)
+  .omit({
+    id: true,
+  })
+  .extend({
+    keys: z.array(z.string()),
```

Comment:
for some reason the createInsertSchema thinks that keys should just be a string zod type so Im overwriting it with extend